controller/cmd/destination/main.go

```diff
@@ -18,18 +19,12 @@ func main() {
 	metricsAddr := flag.String("metrics-addr", ":9999", "address to serve scrapable metrics on")
 	kubeConfigPath := flag.String("kubeconfig", "", "path to kube config")
 	k8sDNSZone := flag.String("kubernetes-dns-zone", "", "The DNS suffix for the local Kubernetes zone.")
-	logLevel := flag.String("log-level", log.InfoLevel.String(), "log level, must be one of: panic, fatal, error, warn, info, debug")
 	enableTLS := flag.Bool("enable-tls", false, "Enable TLS connections among pods in the service mesh")
+	logLevel := logging.LogLevelFlag()
 	printVersion := version.VersionFlag()
 	flag.Parse()
 
-	// set global log level
-	level, err := log.ParseLevel(*logLevel)
-	if err != nil {
-		log.Fatalf("invalid log-level: %s", *logLevel)
-	}
-	log.SetLevel(level)
-
+	logging.SetLogLevel(*logLevel)
```

Comment:
tioi: this exact block is repeated throughout... we could further reduce boilerplate by using an internal `flags` module that sets this all up.

Reply:
That's a great call -- have added a `flags` package in the latest commit and it cleans this up a lot.